src/framing/Frame.cpp

```diff
@@ -94,78 +94,135 @@ std::ostream& operator<<(std::ostream& os, ErrorCode errorCode) {
   return os << "ErrorCode(" << static_cast<uint32_t>(errorCode) << ")";
 }
 
-std::ostream& operator<<(std::ostream& os, FrameFlags frameFlags) {
-  // TODO Match the Flag names with the AllowedFlags in the Frame declarations
+std::ostream& operator<<(
+    std::ostream& os,
+    const std::pair<FrameFlags, FrameFlags>& frameFlagsAndAllowedFlags) {
+  FrameFlags frameFlags = frameFlagsAndAllowedFlags.first;
+  FrameFlags allowedFlags = frameFlagsAndAllowedFlags.second;
+  FrameFlags foundFlags = FrameFlags::EMPTY;
 
   std::stringstream ss;
   std::string delimeter = "";
-  if (!!(frameFlags & FrameFlags::NEXT)) {
-    ss << "NEXT";
+  if (!!(allowedFlags & FrameFlags::IGNORE)
+      && !!(frameFlags & FrameFlags::IGNORE)) {
+    ss << delimeter << "IGNORE";
     delimeter = "|";
+    foundFlags |= FrameFlags::IGNORE;
   }
-  if (!!(frameFlags & FrameFlags::COMPLETE)) {
-    ss << delimeter << "COMPLETE";
+  if (!!(allowedFlags & FrameFlags::METADATA)
+      && !!(frameFlags & FrameFlags::METADATA)) {
+    ss << delimeter << "METADA";
     delimeter = "|";
+    foundFlags |= FrameFlags::METADATA;
   }
-  if (!!(frameFlags & FrameFlags::FOLLOWS)) {
-    ss << delimeter << "FOLLOWS";
+  if (!!(allowedFlags & FrameFlags::RESUME_ENABLE)
+      && !!(frameFlags & FrameFlags::RESUME_ENABLE)) {
+    ss << delimeter << "RESUEN";
     delimeter = "|";
+    foundFlags |= FrameFlags::RESUME_ENABLE;
   }
-  if (!!(frameFlags & FrameFlags::METADATA)) {
-    ss << delimeter << "METADATA";
+  if (!!(allowedFlags & FrameFlags::LEASE)
+      && !!(frameFlags & FrameFlags::LEASE)) {
+    ss << delimeter << "LEASE";
     delimeter = "|";
+    foundFlags |= FrameFlags::LEASE;
   }
-  if (!!(frameFlags & FrameFlags::IGNORE)) {
-    ss << delimeter << "IGNORE";
+  if (!!(allowedFlags & FrameFlags::KEEPALIVE_RESPOND)
+      && !!(frameFlags & FrameFlags::KEEPALIVE_RESPOND)) {
+    ss << delimeter << "KALIVR";
+    delimeter = "|";
+    foundFlags |= FrameFlags::KEEPALIVE_RESPOND;
+  }
+  if (!!(allowedFlags & FrameFlags::FOLLOWS)
+      && !!(frameFlags & FrameFlags::FOLLOWS)) { // FOLLOWS = 0x80,
+    ss << delimeter << "FOLLOW";
+    delimeter = "|";
+    foundFlags |= FrameFlags::FOLLOWS;
+  }
+  if (!!(allowedFlags & FrameFlags::COMPLETE)
+      && !!(frameFlags & FrameFlags::COMPLETE)) { // COMPLETE = 0x40
+    ss << delimeter << "COMPLE";
```

Comment:
I was trying to use short version of these enum values. Let me use the whole word version of them. Seems like it will be problem in the future if I use the short versions.